python/cuml/internals/base_helpers.py

```diff
@@ -125,14 +125,69 @@ def _wrap_attribute(class_name: str,
     return attribute
 
 
+def _check_and_wrap_init(attribute, **kwargs):
+
+    from cuml.common.input_utils import _deprecate_pos_args
```

Comment:
Could be imported at module level

Reply:
This cannot be imported at module level. Circular import. `common` -> `internals` -> `common`. I'm open to a better location or structure to avoid this if you have any ideas.

Reply:
This form of local import is surprisingly expensive, but also technically `_deprecate_pos_args` is not processing inputs, so it would be just as good placed in `internals` and then this can be avoided altogether

Reply:
Moved this. Resolved